.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+**Merge checklist:**
+- [ ] Please do not import models directly in course discovery. e:g if you want to import CourseSkills in Discovery, use the utility get_whitelisted_course_skills instead of directly importing it.
```

Comment:
@zamanafzal @saleem-latif Some thoughts
1. Do we also need to add the rationale behind this? 
2. This rule is hard to enforce because people making taxonomy related changes in course discovery may not be aware of this.
3. Is this rule only applicable to models only?

Reply:
I think this should be removed from here and need to be in developer notes.